### DISCUSSION TITLE
Interesting Project

### DISCUSSION THREAD

**Discussion by ghost** [yanivagman/BPFroid#1] [General]
Recently I learn about ebpf and decided to write a tracer for Android, during searching I found that you already wrote a paper about that and created this tool.
That's amazing!

**Comment by yanivagman** [comment]
Thanks! I'm glad you liked it.
Unfortunately I don't have the time to maintain it,
but maybe in the future I'll merge it back into tracee so it will be better maintained.

**Comment by ghost** [comment]
That would be cool!
